pkgs/top-level/all-packages.nix

```
@@ -6056,6 +6056,8 @@ in
 
   p2pvc = callPackage ../applications/video/p2pvc {};
 
+  p3x-onenote = callPackage ../applications/office/p3x-onenote {};
```

nit, attr set should have a space
```suggestion
  p3x-onenote = callPackage ../applications/office/p3x-onenote { };
```

---

ah thanks done.